tools/travis-test.sh

```diff
@@ -25,7 +25,8 @@ if [ -n "$PYTHON_OPTS" ]; then
 fi
 
 # make some warnings fatal, mostly to match windows compilers
-werrors="-Werror=vla -Werror=nonnull -Werror=pointer-arith"
+# werrors="-Werror=vla -Werror=nonnull -Werror=pointer-arith"
+werrors="-Werror=nonnull -Werror=pointer-arith"
```

Comment:
why do we need this?

Reply:
because ATM HPy uses Variable Length Arrays, so the build fails with "-Werror=vla" on.

Reply:
See discussion in hpyproject/hpy#125.